### DISCUSSION TITLE
访问域名提示 CRYPTO_ERROR (Internal Error) Reason phrase: TLS alert 80

### DISCUSSION THREAD

**Discussion by maxfong** [Tencent/tquic#378] [General]
使用 `tquic-example-c` 工程测试已接入腾讯云服务且已开启 quic 支持的域名  
```
./simple_client wxquicssl.17u.cn 443
```  

报错提示
```
CONNECTION_CLOSE (Transport) Error code: CRYPTO_ERROR (Internal Error)
TLS Alert Description: Internal Error (80)
``` 

Wireshark 截图：  
<img width="943" alt="CONNECTION_CLOSE_CRYPTO_ERROR png" src="https://github.com/user-attachments/assets/637ef800-6d1c-464f-9fb2-d82cb9127ac7">

如何能查到具体的错误点？  感谢

**Comment by iyangsj** [comment]
tquic-example-c/simple_client.c 是仅使用QUIC的简单示例，仅支持 HTTP/0.9 over QUIC（没有支持 HTTP/3）。
在你的场景中，会由于ALPN协商错误而导致握手失败。

如果要使用HTTP/3访问，你可以使用 tquic_client：
- 文档：https://tquic.net/zh/docs/getting_started/demo/
- 代码：https://github.com/Tencent/tquic/blob/develop/tools/src/bin/tquic_client.rs
